scripts/build_bench_precompiles.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Root directory
+SUBNET_EVM_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd .. && pwd
+)
+
+# Load the versions
+source "$SUBNET_EVM_PATH"/scripts/versions.sh
+
+# Load the constants
+source "$SUBNET_EVM_PATH"/scripts/constants.sh
+
+go test ./precompile/contracts/... -bench=./... -covermode=atomic -timeout="10m" $@
```

Comment:
This is great. Can we get the average numbers for all the precompile benchmarks and fail a test if we are significantly over? Currently, we just report, but maybe in the future we can report and compare?

Reply:
ya this would be a good future change to track performance over time